data/hai/unfettered 1 alphas.txt

```diff
@@ -0,0 +1,1532 @@
+			`	This alarms them a lot, and they proceed to a full examination of the bridge and your personal quarters, but to no avail. "Whoever it is that spoke to you must know about our scanning methods. It is very dangerous to proceed with the mission but we have no choice, all we can do is have heavier escorts."`
+			`	Then, almost methodically, she passes the scanner on you, and stops herself, stunned. She then aims her rifle, straight at your shoulder. "Do. Not. Move."`
+			choice
+				`	(Do as told.)`
+				`	(Try to defend yourself.)`
+					goto defend
+			`	She shoots at your shoulder, making you feel dizzy for a while.`
+				goto recovered
+			label defend
+			`	To her surprise, you try to dodge the shot, but all you manage to do is to get yourself stunned by the consecutive ones, all apparently aiming at your shoulder. You almost lose consciousness.`
+			label recovered
+			action
+				clear "midas: tracker relay"
+				"midas: wronged" ++
+			`	Once you recovered, she explains. "Someone had placed a tracker on your shoulder. A very nasty kind of tracker, wired with communications capabilities, and I did not want to find out what else it could do before shooting it. Sorry if I took you by surprise, Captain, but you did well to warn us, and the governor will reward you justly for it once we arrive at the destination."`
+			label leave
+			branch otherships
+				"ships: Space Liner" + "ships: Transport" + "ships: Light Freighter" + "ships: Heavy Freighter" + "ships: Utility" + "ships: Interceptor" + "ships: Light Warship" + "ships: Medium Warship" + "ships: Heavy Warship" >= 1
+			`	The Hai bodyguards now leave your ship, reassured that you are not of any threat for their fleet.`
+				goto admiral
+			label otherships
+			`	The Hai bodyguards now leave your ship, and proceed to check the other ships of your fleet, before reporting a green light.`
+			label admiral
+			`	An hour later, the admiral hails the fleet, "All ships are now ready to leave. All that is left is for me to wish you all safe trip. May the leaves flow our way." You depart with the fleet.`
+				flee
+
+event "unfettered: shipment lost"
+
+
+
+outfit "Building Materials"
+	category "Special"
+	thumbnail "outfit/unknown"
+	cost 1000
+	plural "Building Materials"
+	"mass" 1
+	installable -1
+	description "These appear to be simple building materials, probably meant for the construction of houses. It seems weird to go trough all of this trouble just for these, but it might just be because of the Unfettered's pride."
+
+ship "Bactrian" "Bactrian (Cerberus)"
+	add attributes
+		"outfit space" 17
+		"weapon capacity" 20
+	outfits
+		"Sidewinder Missile Launcher" 2
+		"Sidewinder Missile" 90
+		"Torpedo Launcher" 2
+		"Torpedo" 60
+		"Heavy Laser Turret" 4
+		"Heavy Anti-Missile Turret" 2
+		"Fusion Reactor"
+		"Geode Reactor"
+		"Hai Gorge Batteries"
+		"D67-TM Shield Generator"
+		"Hai Williwaw Cooling" 3
+		"Ramscoop"
+		"Laser Rifle" 20
+		"Fragmentation Grenades" 20
```

Comment:
It’s a pirate ship, so,
```suggestion
		"Fragmentation Grenades" 20
		"Nerve Gas" 200
```
Maybe a bit less, but still.

Reply:
An alpha ship, even
But you're supposed to be able to capture it (except if this is the other variant)

Reply:
oh wait this is the Cerberus ship not the Alpha ship, good point then but no more than 15 caus its pricy

Reply:
> An Alpha ship, even
> But you're supposed to be able to capture it (except if this is the other variant)

Nerve Gas doesn’t have a great defense. 
They maybe shouldn’t have any because no other ships in the game data have it, but…

Reply:
I got another ship that has some (of the rival faction though)

Reply:
Your choice on whether it should use.

Reply:
Resolved in 3800bed.